src/pulp/pulp.py

```diff
@@ -1668,7 +1668,7 @@ def solve(self, solver = None, **kwargs):
         return status
 
     def sequentialSolve(self, objectives, absoluteTols = None,
-                        relativeTols = None, solver = None, debug = False):
+                        relativeTols = None, solver = None, debug = False, timeout = None):
```

Comment:
I would be happer is we used **kwargs to pass solver parameters here

Reply:
@bertrand-caron Sounds reasonable, right?

Reply:
@stumitchell Using kwargs in all methods, i.e. also in solve_CBC?